avocado/plugins/spawners/podman.py

```diff
@@ -86,8 +87,84 @@ def is_task_alive(self, runtime_task):
         # FIXME: check how podman 2.x is reporting valid "OK" states
         return out.startswith(b'Up ')
 
+    def _fetch_avocado_egg(self, python_major, python_minor):
```

Comment:
I guess post-draft there will be `_fetch_python_egg("avocado", ...)`  and some less hardcoding but I understand your idea now.